### ISSUE TITLE
error when creating randomized charachter

### ISSUE THREAD

**Issue by yoopgim** [CleverRaven/Cataclysm-DDA#59968]
### Describe the bug

I try to use the "play now" option under new game in the main menu and when its loading the data for whatnot it has an error when loading the files for the core game (i do have mods installed and while i highly doubt it has anything to do with it i will mention them if anyone needs to know while fixing this) and creates an error report then puts me back on the main menu.

### Steps to reproduce

I press the play now option under the new game option in the main menu then wait.

### Expected behavior

it gets through the loading process and puts me in the game

### Versions and configuration

i am using cdda-windows-tiles-sounds-x64-2022-08-06 (P.S. what is the difference between "msvc" versions and other versions?)

### Additional context

screenshot of error report:
![Screenshot 2022-08-06 105310](https://user-images.githubusercontent.com/51343338/183256413-66e19df7-e079-4f27-8e79-fcc664fd07f3.png)

**Comment by Night-Pryanik** [comment]
Please post your mods.

**Comment by yoopgim** [comment]
ok mod order is this: 
![Screenshot 2022-08-06 110815](https://user-images.githubusercontent.com/51343338/183256945-15758ec0-d04c-4a56-b45b-d87c18299f2d.png)

**Comment by yoopgim** [comment]
i also just tested it in a world with just the core game and no mods (even the mods enabled by default on download) and it has the same issue and error report

**Comment by BrettDong** [comment]
Did you try downloading the game from GitHub and decompressing to a clean folder?

**Comment by yoopgim** [comment]
no i downloaded it from the site and put my old saves into it, which might have been the problem. i'll try doing that if i can figure out how.

**Comment by yoopgim** [comment]
ok how do i download it from github?

**Comment by EIIKaO** [comment]
download from github.com/CleverRaven/Cataclysm-DDA/releases here
https://github.com/CleverRaven/Cataclysm-DDA/releases

I suspected it was due to you **full** of overriding 2022-08-06 to/from an existing game folder (I just had a similar problem).

Because the file `monster_gun.json` (include "barb_launcher") was moved/split about 10 days ago, so if you have **totally** overridden to/from the old game folder, you have duplicate file.

I also recommend that you try a clean install as BrettDong mentioned.
In addition, another trial temporary option would be here, rename the file (If you have file management skill). 

Rename from `data\json\items\monster_gun.json` ( if exists ) to `data\json\items\monster_gun.json_`

I think the problem can be solved If it was due to the first thing I mentioned.
But there may be other problems **I recommend that finally you do a clean install and then copy over only the save (and neccesary mods) from the old game foler**).
